OpenEdXMobile/src/main/java/org/edx/mobile/view/BaseWebViewFragment.java

```diff
@@ -9,8 +9,8 @@
 import android.webkit.WebResourceRequest;
 import android.webkit.WebResourceResponse;
 import android.webkit.WebView;
-import android.widget.ProgressBar;
 
+import com.google.android.material.progressindicator.CircularProgressIndicator;
```

Comment:
I think we don't need these files changes.

Reply:
update the references for code improvements.